R/geom-boxplot.r

```diff
@@ -204,7 +207,7 @@ GeomBoxplot <- ggproto("GeomBoxplot", Geom,
         size = outlier.size %||% data$size[1],
         stroke = outlier.stroke %||% data$stroke[1],
         fill = NA,
-        alpha = NA,
+        alpha = outlier.alpha %||% data$alpha[1],
```

Comment:
`%||%` works with `NULL` not  `NA`. I think that just means the default function param above should be NULL.